basics/list-comprehension.md

```diff
@@ -0,0 +1,77 @@
+# List Comprehension
+
+
+## Definition
+List comprehensions creates a new list from an existing list that is a shorter syntax than normal. It also processes the list much faster than using a for loop.
```

Comment:
- "Shorter than normal" is a bit ambiguous.
- When performance matters, performance problems don't usually come from small things like this, in my experience. Not really worth mentioning.

```suggestion
List comprehensions creates a new list from an existing list using shorter syntax than loops.
```